RecoHGCal/TICL/plugins/PatternRecognitionbyCA.h

```diff
@@ -53,5 +53,9 @@ namespace ticl {
 
     static const int eidNFeatures_ = 3;
   };
+
+  template class PatternRecognitionbyCA<TICLLayerTiles>;
```

Comment:
These should not be declared in the .h file. They should be in the .cc. We want them 'created' only once, not in each file that includes this header.